src/versions.js

```diff
@@ -0,0 +1,50 @@
+// @ts-check
+
+// Note: these type definitions are only checked within an editor. We aren't set up
+//   for type-checking in the build (yet!) but having these types defined helped me write this code.
+/**
+ * @typedef {object} VersionMapping
+ * @property {string} docsVersion
+ * @property {string} optimizeVersion
+ *
+ * @typedef {object} UnsupportedVersion
+ * @property {string} label
+ * @property {string} urlSuffix
+ */
+
+/** @type {Array<VersionMapping>} */
+const versionMappings = [
+  // 👋 When cutting a new version, add a new mapping here!
+  {
+    docsVersion: "8.3",
+    optimizeVersion: "3.11.0",
+  },
+  {
+    docsVersion: "8.2",
+    optimizeVersion: "3.10.0",
+  },
+  {
+    docsVersion: "8.1",
+    optimizeVersion: "3.9.0",
+  },
+  {
+    docsVersion: "1.3",
+    optimizeVersion: "3.7.0",
+  },
+];
+
+/** @type {Array<UnsupportedVersion>} */
+const unsupportedVersions = [
+  // 👋 When archiving a version, move it from the above array into here, and edit it!
+  { label: "8.0 / 3.8.0", urlSuffix: "8.0" },
```

Comment:
Since we didn't implement the first version introducing Optimize docs to the archive (yet), I wasn't sure what this line implies. ([Similar line here](https://github.com/camunda/camunda-docs/pull/2502/files#diff-fa6812b60b6bdb23ffa969f6579e342186e072181dc52c0ebf6d1fa8c01d6022R35).)

Will this add an 8.0 to the URL for all the docs? Including Optimize? You may have explained this already, but I'm having trouble finding it.

Reply:
The `label` is what will appear in the dropdown in the top nav; the `urlSuffix` is what will be appended to `https://unsupported.docs.camunda.io/` in the link. So for this unsupported version, the link would take you to `https://unsupported.docs.camunda.io/8.0/` -- which is where both the main 8.0 docs and optimize 3.8.0 docs will be living. 

And once on that unsupported/8.0 site, the main docs live at `unsupported.docs.camunda.io/8.0/docs/path/to/page`, and the optimize docs live at `unsupported.docs.camunda.io/8.0/optimize/path/to/page`.

I'm not exactly sure which part of that explanation is most unclear, so if there is something I could add comment-wise to clarify, I'm up for it.

Reply:
Is this a typo in the description? I think this is what I'm getting hung up on more than the line of code itself.

> Redirect all 8.0 / 3.8.0 traffic to unsupported.docs.camunda.io/1.3 (or stage.unsupported.docs.camunda.io/8.0 for the staging build)

This should be `unsupported.docs.camunda.io/3.8`? 

The application of the `urlSuffix` could be clearer, just as you've described here. However, I'm wondering if there is a typo on the Optimize URL. 

`unsupported.docs.camunda.io/8.0/optimize/path/to/page`

vs.

`unsupported.docs.camunda.io/3.8/optimize/path/to/page`

While I don't think it will be a point of contention since it's an unsupported version, we should be clear that the URL for Optimize will include a version number for the Camunda platform version it's associated with.

Reply:
> 👍 I'll try to add comments to clarify.

Done, in my latest commit!

Reply:
Also @akeller you can see how the main docs & optimize instances interplay re: versions and URLs at https://unsupported.docs.camunda.io/8.0/.